apps/ui/src/helpers/connectors/sequence-waas.ts

```diff
@@ -0,0 +1,48 @@
+import Connector from './connector';
+
+const projectAccessKey = 'AQAAAAAAAKD5WUrmbwO_EbPcAST_dV5PpDg';
+
+export default class SequenceWaaS extends Connector {
```

Comment:
I wouldn't add WaaS in the name
```suggestion
export default class Sequence extends Connector {
```